packages/@vivliostyle/theme-base/css/common/meta-properties.css

```
@@ -0,0 +1,56 @@
+:root {
+  /**
+   * Typography
+   */
+  --vs-font-family: sans-serif;
+  --vs-font-feature-settings: 'kern', 'liga', 'calt';
```

'liga' と 'calt' はデフォルトでonになってるので、不要でないでしょうか。

また、'kern' は font-kerning プロパティで制御するべきでしょう。
font-kerning プロパティの初期値 auto で、どのブラウザでも欧文のカーニングが有効になります。
日本語文字のカーニングについては、デフォルトでは有効にするべきではなく、また、'kern' を有効にする場合には 'palt' （縦書きでは 'vpal'）も有効にするべきだという議論があり、すでにFirefoxではそうなってます。
（デフォルトの font-kerning: auto では日本語文字の部分の 'kern' は off で、 font-kerning: normal を指定すると、'kern' と 'palt/vpal' が on になる。
font-feature-settings: 'kern' の指定によっても、'palt/vpal' が on になることに注意！）

参照 https://bugzilla.mozilla.org/show_bug.cgi?id=1797431


```suggestion
  --vs-font-feature-settings: normal;
```

---

デフォルトのfont-feature-settingsを `normal` に変更しました。合わせて `--vs-font-kerning` の設定を追加しました https://github.com/vivliostyle/themes/blob/c70e5fe79f972b34a17d390a22107052ae69eeda/packages/%40vivliostyle/theme-base/css/common/meta-properties.css#L6-L7